src/antsibull_core/vendored/collections.py

```diff
@@ -103,7 +103,7 @@ def count(seq):
     deprecated and replaced when support for Python < 2.7 is dropped.
     """
     if not is_iterable(seq):
-        raise Exception('Argument provided  is not an iterable')
+        raise RuntimeError('Argument provided  is not an iterable')
```

Comment:
I didn't touch this before, as it's vendored code. If you do want to change this, it should be a `TypeError`.

Reply:
Oh right. Let's just ignore it...